File: gif.js
const fetch = require("node-fetch");

module.exports = async function (msg, args) {
  let keywords = "love";
  if (args.length > 0) {
    keywords = args.join(" ");
  }
  let url = `https://api.tenor.com/v1/search?q=${keywords}&key=${process.env.TENORKEY}&contentfilter=high`;
  let response = await fetch(url);
  let json = await response.json();
  const index = Math.floor(Math.random() * json.results.length);
  msg.channel.send(json.results[index].url);
};

File: gif.py
import requests
import json
import random
from datetime import datetime
random.seed(datetime.now())


def GIF(message):
    if len(message) == 4:
        message = "$gif love"
    if message[4] != " ":
        return "Command Not found: Please add a space after $gif"
    url = 'https://api.tenor.com/v1/search?q='+ message[5:] +'&key=SEEQLCR7IAO4&contentfilter=high'
    r = requests.get(url)
    data = r.json()
    results = data["results"]
    index = random.choice(range(0, len(results)))
    return results[index]["url"]

